### ISSUE TITLE
Replace NEWS and MAIL with other meaningful alternatives that fit this century.

### ISSUE THREAD

**Issue by tsathoqqua** [limalib/lima#10]

**Comment by azerwyn** [comment]
What we have today is a product of necessity from long ago. One could argue, the mud news and mud mail munged real world social interaction with in-game/in-character interaction.  Communication channels delivered or coordinated game administrivia while also providing stickiness of being a part of something, being needed, and satisfying the need to be needed.

To get off the blank page, I think the best thing we can do is to challenge old assumptions and ask questions.

- Given the abundance of social media available is there value or a need to (re)invent the wheel?
- Are there API opportunities to leverage best-in-class products?
- Given the much smaller audience for this medium, is IMud living up to it's potential?

While we are asking this question for Lima, is the larger community also facing similar modernity challenges? This last question I separate out because it feels more like a brainstorm question than one with an easy or definitive answer.

**Comment by salterryanjr** [comment]
Discord integration with channels for news and stuff?

**Comment by tsathoqqua** [comment]
You're right on the money. We need to consider how the world changed since 1996. I think a lot of things that were assumed to be necessary in 1996, are no longer needed. I'm fighting already to keep the bug reports for LIMA in _1_ place and not spread over Discord servers, PMs, Github and over a cup of coffee with my friends. 

I think the NEWS in LIMA could easily be replaced by other more modern mechanisms. I think most MUDs have Discords alongside their games where things that would originally be discussed in MUDs with have moved to.

News replacement? Perhaps a simpler message board with information from admins about events? We already have a bulletin board - could they replace it, be made better?

MAIL, on the other hand, is different. I think it comes in two parts:

1. In game mail that are sent from an NPC or to make you aware of forgotten loot or items. Most MMOs can send you a mail with loot you didn't pick up or claim, or make you aware of things happening in the game.
2. Private PMs between players, boyfriends/girlfriends, "when we do we run the Demon pit, Bob?"

No. 1: We cannot really do today, we have no way of "attaching items" today, should we have? 
No. 2: Probably already handled in discord, not needed in game? Okay to support, but not the primary use case.

I've been looking into using things like 'curl' to post issues directly to Github, and we might actually make things like the 'bug' 'idea' and other commands that we have today post directly to an issue on Github.

**Comment by tsathoqqua** [comment]
Oh, and side note. I'd like to create an auction house for selling and buying valuable items/uniques rather than people posting to NEWS about how they have "a dragon slayer +5 of doom for sale for 50.000 gold".
